### ISSUE TITLE
Tawhid Series – Usul Al-Thalathah: Part 1 – Introduction

### ISSUE THREAD

**Issue by ikhbatalqulub** [ikhbatalqulub/Detailed-lecture-summaries#2]
Please describe the specific content (outline the content) of this lecture with around 500 characters (including spaces).
https://www.youtube.com/watch?v=qtnt93Da0r0

**Comment by deeos** [comment]
Can this be done in list format?

**Comment by deeos** [comment]
It’s difficult to describe things in 500 characters

**Comment by aghamidi** [comment]
It can be in list format, but it should be organised and exhaustive.

This is the task. It should be around 500 characters.

**Comment by abuzgezgoz** [comment]
It is a booklet not book on Tawheed, it smaller in words but it has great meanings which no one can be ignorant of. Scholars have always discussed this book and taught it among the first to be taught in Tawheed. The three fundamental principles (Usul Al-Thalathah) in reality are amongst what you are going to be asked in the grave.

**Advise on seeking knowledge by shaykh Ahmad Jibril:**
The shaykh (Ahmad Jibril) will start gradually as he advises from Ibn Abdi AL-Barr book Al-Jamii that Azzuhr said “whoever seeks to get knowledge in a lump sum (or overnight), knowledge ends up leaving him in a lump sum”. Knowledge needs patience and persistence.
It is not a good practice to limit ourselves by reading books only when seeking knowledge. We should try our best to learn with the shaykhs in classes, that is the best way to seek knowledge.
We should jot down whatever we can while seeking knowledge.
Going to the book now…….
**BISMILLAH ARRAHMANI ARRAHIM**
As student of knowledge we should understand the meaning of Bismillah Arrhamani Arrahim not like a regular person. We should get the entire perspective as to why we saying Bismillah Arrahman Rrahim to everything. When you saying Bismillah Arrahman Arrahim, you are exerting your entire Tawheed to ALLAH in that matter in which you are saying Bismillah to. It has all aspects of Tawheed in it:

**a.	Tawheed Uluhiyya:**
By saying Bismillah, you are saying:
-	Allah permits me to do this – That’s why you don’t say Bimillah when starting a sinful deed
-	I am doing this for the sake of ALLAH 

**b.	Tawheed Rubbubiyyah:**
By saying Bismillah You admit that:
-	You couldn’t not have done whatever you are doing without Allah`s power to give you the ability to do it, that is why you say Bismillah. It is like saying la hawla wala quwatta illa billah ("There is no might nor power except in Allah")

**c.	Tawheed AL-Asmaa Wa Sifaat:**
By saying Bimillah:
-	You are using the name of ALLAH to bless the act you are doing.

 **Linguistic note on Bismillah Arrahmani Arrahim:**
The letter “BA” in Bimsillah is istiannah and tawakkul (to seek help in dependence on ALLAH)
When you say Bismillah in Arabic, it automatically implies that you are saying Bismillah to the act you are about to do i.e. you don’t have to specify “Bismillah I drink, I eat. I write etc.” Bismillah is sufficient. 
In other words, The Arabic language eliminates the need to mention the act you are saying Bismillah to, it is automatically implied.

**Why does the author of the book Usul Al-Thalathah starts with Bismillah**
a.	The author uses proof from Quran, that’s why he uses Bismillah like in the Quran
b.	It was the tradition of the prophet to start letters with Bismillah Arrahman Arrahim
c.	There is a hadith that any important matter without Bismillah it keeps away good (ref?)
d.	To bless his work

**Comment by deeos** [comment]
This is the start of the explanation of the three fundamental principles, a booklet on tawheed which addresses the three questions we will all be asked in the grave.   

Contents:
Introduction
Islamic knowledge is to be taken gradually 
Patience and persistence upon knowledge
Classical study
Three levels of studying
Writing down knowledge 
Usul al thalath 
The basmalah
Tawheed in the basmalah
Linguistics rules in the basmalah
Proof for beginning with the basmalah
The blessing in the basmalah

**Comment by aghamidi** [comment]
@deeos @abuzgezgoz 
Brothers, I think we will cancel the summaries for now. There isn't enough interest and I think providing you with feedback will be too consuming at the moment because the above is not really what we are looking for. Barak Allah feekum for your efforts and I apologise for the inconvenience.

**Comment by abuzgezgoz** [comment]
No worries akhi,,,,... if you feel there are easier tasks fit for me do let me know... that is the best I can do, How I wish I could help....May ALLAH reward you for your work..AMEEN!!!

**Comment by deeos** [comment]
No problem, wa feeka Barak Allah

**Comment by ikhbatOfficial** [comment]
@abuzgezgoz We haven't uploaded the lectures yet. We will upload them to youtube and our website ikhbat.com in shaa Allah. When we launch the website, I will announce it on my Facebook and Twitter bi idhniLah.
